doc/source/installation-dev.rst

```diff
@@ -0,0 +1,13 @@
+.. _installation-dev:
+
+Installation for DEV's
+======================
+
+``pyramid_oereb`` is a piece of python software following the rules of the pyramid webframework. To develop
+features there are basically two ways to setup:
+
+# Docker/Docker-Compose
+# local system python with a virtualenv
```

Comment:
this is not rendered correctly in my .html. : 
# Docker/Docker-Compose # local system python with a virtualenv

Reply:
Hey Anne. Thank you for checking out. This PR is meant to fix the structure to be able to build the DOC again. The content will be fixed afterwards.